files/puppet_helper.groovy

```diff
@@ -170,10 +170,13 @@ class Actions {
       )[0].getStore()
 
     def credentials
+    if (id == "") {
+      id = null
+    }
```

Comment:
From looking at the existing code, I thought that the cli argument to method call conversion, `actions."$action"(*args[1..-1])`, must be requiring string params to be defaulted to an empty string.  However, I just tested it and there doesn't appear to be an issue with defaulting a string param to `null`.  

The `id` param could be defaulted to `null` and then this check wouldn't be needed.